packages/studiocms_core/src/schemas/config/developer.ts

```diff
@@ -13,7 +13,7 @@ export const developerConfigSchema = z
 		/**
 		 * Enable Testing and Demo Mode
 		 *
-		 * This will enable the testing and demo mode for the Astro Studio CMS Dashboard, this will allow you to test the dashboard without having to authenticate. This is useful for testing and demo purposes as it will allow you to see how the dashboard works and looks but disable any changes to the database.
+		 * This will enable the testing and demo mode for the Astro StudioCMS dashboard, this will allow you to test the dashboard without having to authenticate. This is useful for testing and demo purposes as it will allow you to see how the dashboard works and looks but disable any changes to the database.
```

Comment:
This is still using the old name

Reply:
fixed as of [aae4d48](https://github.com/astrolicious/studiocms/pull/352/commits/aae4d48b71e3d860774b1789626ff4bb1c6c1641)